store/file/ods.go

```diff
@@ -228,28 +228,28 @@ func (o *ODS) Close() error {
 // Sample returns share and corresponding proof for row and column indices. Implementation can
 // choose which axis to use for proof. Chosen axis for proof should be indicated in the returned
 // Sample.
-func (o *ODS) Sample(ctx context.Context, rowIdx, colIdx int) (shwap.Sample, error) {
+func (o *ODS) Sample(ctx context.Context, idx shwap.SampleIndex) (shwap.Sample, error) {
 	// Sample proof axis is selected to optimize read performance.
 	// - For the first and second quadrants, we read the row axis because it is more efficient to read
 	//   single row than reading full ODS to calculate single column
 	// - For the third quadrant, we read the column axis because it is more efficient to read single
 	//   column than reading full ODS to calculate single row
 	// - For the fourth quadrant, it does not matter which axis we read because we need to read full ODS
 	//   to calculate the sample
-	axisType, axisIdx, shrIdx := rsmt2d.Row, rowIdx, colIdx
-	if colIdx < o.size()/2 && rowIdx >= o.size()/2 {
-		axisType, axisIdx, shrIdx = rsmt2d.Col, colIdx, rowIdx
-	}
+	axisType, axisIdx := rsmt2d.Row, 0 // rowIdx, colIdx
+	// if colIdx < o.size()/2 && rowIdx >= o.size()/2 {
+	// 	axisType, axisIdx, shrIdx = rsmt2d.Col, colIdx, rowIdx
+	// }
```

Comment:
we still need to swap coordinates here.

Reply:
Looks like we don't need `rowIdx, colIdx` (`idx` is enough here). The only question I have is: how to get from `SampleIndex` which axis do we want?

Reply:
Fixed